core/src/io/github/teamfractal/entity/Market.java

```diff
@@ -205,7 +198,15 @@ void setResource(ResourceType type, int amount)
     boolean hasEnoughResources(ResourceType type, int amount)
             throws InvalidResourceTypeException {
         int resource = getResource(type);
-        return amount <= resource;
+        /**
+         * Added by JBT to fix crash when resource and amount = 0
```

Comment:
/** */ is reserved for docstrings. // is for a single line comment. /* */ for multi line comments